widgets/common/src/deploymentsView/detailsPane/drilldownButtons/ParentButton.tsx

```diff
@@ -0,0 +1,25 @@
+import { useMemo } from 'react';
+import type { FunctionComponent } from 'react';
+import { tDrillDownButtons } from './common';
+import { isTopLevelPage } from '../../common';
+import DrilldownButton from './DrilldownButton';
+
+interface ParentButtonProps {
+    toolbox: Stage.Types.Toolbox;
+}
+
+const { Icon } = Stage.Basic;
+const tParentButton = Stage.Utils.composeT(tDrillDownButtons, 'parent');
+
+const ParentButton: FunctionComponent<ParentButtonProps> = ({ toolbox }) => {
+    const drilldownContext = ReactRedux.useSelector((state: Stage.Types.ReduxState) => state.drilldownContext);
+    const showParentButton = useMemo(() => !isTopLevelPage(drilldownContext), [drilldownContext]);
+
+    return showParentButton ? (
+        <DrilldownButton onClick={() => toolbox.goToParentPage()} title={tParentButton('title')}>
+            <Icon name="angle left" />
+            {tParentButton('label')}
```

Comment:
WDYT about using the `t` variable name instead of `tParentButton`? 🤔 

As we are only using this function inside of `ParentButton`, we don't have a necessity of making its name even more related to the `ParentButton` 😁 

In addition, it would make our application more consistent as we are mostly using this approach 🚀 
(I've found out `2892` occurences of its usage, via IDE search 😁)

```suggestion
const t = Stage.Utils.composeT(tDrillDownButtons, 'parent');

const ParentButton: FunctionComponent<ParentButtonProps> = ({ toolbox }) => {
    const drilldownContext = ReactRedux.useSelector((state: Stage.Types.ReduxState) => state.drilldownContext);
    const showParentButton = useMemo(() => !isTopLevelPage(drilldownContext), [drilldownContext]);

    return showParentButton ? (
        <DrilldownButton onClick={() => toolbox.goToParentPage()} title={t('title')}>
            <Icon name="angle left" />
            {t('label')}
```

Reply:
I used the same approach as in other buttons in Deployments View widget, but OK, I'll change it.